android/library/src/main/java/io/logto/android/authflow/webview/WebViewAuthFlow.kt

```diff
@@ -10,14 +10,18 @@ import io.logto.android.client.LogtoClientBuilder
 import io.logto.android.client.api.LogtoClient
 import io.logto.android.constant.AuthConstant
 import io.logto.android.model.Credential
+import io.logto.android.storage.CredentialStorage
 import io.logto.android.utils.PkceUtil
 import io.logto.android.utils.UrlUtil
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 
-class WebViewAuthFlow {
+class WebViewAuthFlow(
+    private val context: Context,
+    private val credentialStorage: CredentialStorage?
+) {
 
     private lateinit var logtoConfig: LogtoConfig
     private lateinit var logtoClient: LogtoClient
```

Comment:
I'm wondering if we can combine these into the same namespace, to keep it simpler. say `io.logto.android.Abc` for most common modules that user will use, such as `io.logto.android.Config` and `io.logto.android.Client`

Reply:
more generally speaking, `android` can be removed on those non platform-related modules, such as `Config`, maybe `Client` as well if the PKCE flow doesn't use any Android-specific technique. Let's create a Linear issue for this and add the eng team for subscription. I'll review the whole design once I'm available.

Reply:
* [LOG-95: Move LogtoConfig & LogtoClient into separate modules](https://linear.app/silverhand/issue/LOG-95/move-logtoconfig-and-logtoclient-into-separate-modules)
* [LOG-96: Android SDK architecture design](https://linear.app/silverhand/issue/LOG-96/android-sdk-architecture-design)